shared/analytics/track.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+const debug = require('debug')('api:analytics:track');
+import Raven from 'shared/raven';
+import { amplitude } from './amplitude';
+
+export const track = (
+  userId: string,
+  eventType: string,
+  eventProperties: Object = {}
+) => {
+  const amplitudePromise = () => {
+    debug(`[Amplitude] Tracking ${eventType}`);
+    return amplitude.track({
+      userId,
+      eventType,
+      eventProperties: {
+        ...eventProperties,
+        client: 'api',
```

Comment:
Should this be set per-consumer? If it's tracked in athena we should set this to `athena`?

If so, you can get access to the current servers' name via `process.env.SENTRY_NAME`. (which is set to the folder name of the server in `backpack.config.js`)